lxd/db/generate/db/mapping.go

```diff
@@ -273,6 +273,25 @@ func (f *Field) Column() string {
 	return column
 }
 
+// ScalarTableColumn gets the table and column from the join configuration.
+func (f Field) ScalarTableColumn() (string, string, error) {
+	join := f.Config.Get("join")
+	if join == "" {
+		join = f.Config.Get("leftjoin")
+	}
+
+	if join == "" {
+		return "", "", fmt.Errorf("Missing join config")
```

Comment:
Will this error get wrapped with the field's name or should we add it here, so that if this failure occurs we know which field its for.

Reply:
It's not wrapped so I'll add it here.